flight/Modules/PathPlanner/pathplanner.c

```diff
@@ -253,6 +250,7 @@ static void holdCurrentPosition()
 	pathDesired.StartingVelocity = 5; // This will be the max velocity it uses to try and hold
 	pathDesired.EndingVelocity = 5;
 	pathDesired.ModeParameters = 0;
+	pathDesired.Waypoint = -1;
```

Comment:
It could be important to avoid skipping waypoint 1 if you were previously on waypoint 1.  (Eliminates a minor race).  I agree with holdLastPosition-- willfix.